### DISCUSSION TITLE
`Sse` + `TraceLayer` `on_eos`?

### DISCUSSION THREAD

**Discussion by GabrielBianconi** [tokio-rs/axum#2593] [Q&A]
### Summary

I have a handler that streams an `SSE` response. I'm trying to setup logging with `TraceLayer`, but noticed that Axum is not triggering `on_eos` when the SSE stream ends. Am I misunderstanding how this works?

### axum version

0.7.4

**Comment by KyleSebion** [comment]
I'm having a similar problem. axum version 0.8.4.

**Comment by mladedav** [comment]
You're most likely running into [this issue](https://github.com/tower-rs/tower-http/issues/550).
